docs/user/rss/flow/input/input.md

```diff
@@ -0,0 +1,225 @@
+(rss)=
+
+# Input YAML configuration file
+
+The YAML configuration file contains all the inputs you need to define and customize your RSS workflow. This section provides a detailed explanation of how to set up the file, using silicon (`Si`) as an example. To make it easier to understand, we will break down the YAML file into several sections based on the key components of the RSS workflow and explain each part in detail.
+
+## General parameters
+
+The section defines the general settings for the RSS workflow, including the system's formula, whether to start fresh, or resume from a previous state.
+
+```yaml
+# General Parameters
+tag: 'Si'
+train_from_scratch: false
+resume_from_previous_state:
+  test_error:
+  pre_database_dir:
+  mlip_path:
+  isolated_atom_energies:
+```
+
+The `tag` identifies the elements in the system, such as `Si` in this example, and can also be passed to the parameters for random structure generation. Setting `train_from_scratch=false` indicates that the RSS workflow will start from scratch. To resume a workflow, refer to the [Resuming workflow from point of interruption](../quick_start/start.md) section.
+
+## Buildcell parameters
+
+The section defines the settings for generating initial random structures. These parameters control the diversity, symmetry, size, and number of the generated structures. We utilize the `buildcell` software from [AIRSS](https://airss-docs.github.io/technical-reference/buildcell-manual) for random structure generation, ensuring flexibility and efficiency in exploring configurational space.
+
+```yaml
+# Buildcell Parameters
+generated_struct_numbers:
+  - 8000
+  - 2000
+buildcell_options:
+  - NFORM: '1'
+    SYMMOPS: '1-4'
+    SLACK: 0.25
+    OVERLAP: 0.1
+    NATOM: '{6,8,10,12,14,16,18,20,22,24}'
+  - NFORM: '1'
+    SYMMOPS: '1-4'
+    SLACK: 0.25
+    OVERLAP: 0.1
+    NATOM: '{7,9,11,13,15,17,19,21,23}'
+fragment_file: null
+fragment_numbers: null
+num_processes_buildcell: 128
+```
+
+The `buildcell_options` parameter is one of the most critical settings, as it determines the scope of the RSS search and directly influences the diversity of the generated structures. We provide multiple sets of `buildcell` input parameters. In the configuration above, 8,000 even-numbered structures and 2,000 odd-numbered structures will be generated, with symmetry operations ranging from 1 to 4. You can supply a single set or multiple sets as needed. This flexibility ensures that the initial structures have sufficient diversity. In principle, any parameter supported by [buildcell](https://airss-docs.github.io/technical-reference/buildcell-manual) can be used in the `buildcell_options` section.
```

Comment:
Hi @YuanbinLiu , general question `8000 even-numbered structures and 2000 odd-numbered` this phrase refers to total number of atoms in structure is even and odd? Sounds a bit unclear and purpose is not clear. Maybe this takes care of different Stoichiometry explored?

Maybe slightly repharse it to something like the one below ?

Using the configuration file above, a total of 8,000 structures will be generated, each containing an even number of atoms. Additionally, 2,000 structures will be created with an odd number of atoms. The symmetry operations will vary from 1 to 4.

Reply:
@naik-aakash  this sounds better! Thanks

Reply:
done!